libraries/AP_RCProtocol/AP_RCProtocol_DSM.cpp

```diff
@@ -17,7 +17,12 @@
 /*
   with thanks to PX4 dsm.c for DSM decoding approach
```

Comment:
this no longer has any PX4 code with your change, so we can remove this comment now